crawl-ref/docs/options_guide.txt

```diff
@@ -985,9 +985,15 @@ auto_switch = false
         weapon when firing or attacking in melee, as long as the one you are
         wielding and the one you switch to are both in slot 'a' or 'b'.
 
-travel_open_doors = true
-        If this is set to false, autoexplore/travel will not open doors,
-        instead stopping in front of them.
+travel_open_doors = (avoid | approach | open)
+        Change how autoexplore/travel interact with closed doors.
+           avoid = If they can explore (or reach their destination) without
+                   opening doors, they will do. If not, the stop by a closed
+                   door.
+        approach = They walk up to a door if it's on their route, and stop in
+                   front of it.
+            open = They open any closed doors in their path. (default)
+        This does not affect runed door, which they always avoid.
```

Comment:
```suggestion
travel_open_doors = (avoid | approach | open)
        Configure how autoexplore/travel interacts with doors.
           avoid = Autoexplore/travel will treat closed doors like walls. If the only
           way for exploration or travel to continue is through a closed door, it will stop by
           a closed door.
        approach = Autoexplore/travel will not open doors, instead stopping in front of any
        door in its path.
            open = Autoexplore/travel will open doors. (default)
        This does not affect runed doors, which are always avoided.
```

I can't set the whitespace correctly in GitHub, but this is some improved wording.